From bc9d42832325bc638802ca66e29f9720f6c37ada Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <uwu@icenowy.me>
Date: Sun, 25 Sep 2022 23:02:39 +0800
Subject: [PATCH 42/97] arm64: add Kconfig option for Phytium SoCs

This option works as a gate for Phytium-specific drivers.

Signed-off-by: Icenowy Zheng <uwu@icenowy.me>
Ref: Contact the author.
---
 arch/arm64/Kconfig.platforms | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 6c6d11536b42..fb762003b209 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -262,6 +262,11 @@ config ARCH_PENSANDO
 	  cores for a minimal latency/jitter datapath, and network
 	  interfaces up to 200 Gb/s.
 
+config ARCH_PHYTIUM
+	bool "Phytium Platforms"
+	help
+	  General support for ARMv8 based Phytium SoCs.
+
 config ARCH_QCOM
 	bool "Qualcomm Platforms"
 	select GPIOLIB
-- 
2.46.0

